src/types/Account/Account.js

```diff
@@ -61,7 +61,7 @@ class Account extends EventEmitter {
     this.injectDefaultPlugins = _.has(opts, 'injectDefaultPlugins') ? opts.injectDefaultPlugins : defaultOptions.injectDefaultPlugins;
     this.allowSensitiveOperations = _.has(opts, 'allowSensitiveOperations') ? opts.allowSensitiveOperations : defaultOptions.allowSensitiveOperations;
     this.debug = _.has(opts, 'debug') ? opts.debug : defaultOptions.debug;
-    if (this.debug) process.env.LOG_LEVEL = 'debug';
+    // if (this.debug) process.env.LOG_LEVEL = 'debug';
```

Comment:
This makes the 'debug' property useless. May be we should remove it.

Reply:
Yeah, let's fix it in the future release due to lack of time